docs/integrations/app-integrations/microsoft-teams.mdx

```diff
@@ -3,189 +3,101 @@ title: Microsoft Teams
+  <Tip>
+  If you already have an existing app registration, you can use it instead of creating a new one and skip steps 4-6.
+  </Tip>
+
+  8. Open the MindsDB Editor and create a connection to Microsoft Teams using the client ID and client secret obtained in the previous steps using the SQL command provided above.
+  9. Using this connection, create a chatbot using the [`CREATE CHATBOT`](/agents/chatbot) syntax.
+  10. Run the `SHOW CHATBOTS` command and record the `webhook_token` of the chatbot you created.
+  11. Navigate back to the Microsoft Bot Framework portal and fill out the messaging endpoint in the following format: `<mindsdb_url>/api/webhooks/chatbots/<webhook_token>`.
+  <Tip>
+  The `<mindsdb_url>` is the URL of where MindsDB is running.
+  Please note that if you are running MindsDB locally, it will need to be exposed to the internet using a service like [ngrok](https://ngrok.com/).
+  </Tip>
+
+  12. Fill out the *Microsoft App ID* using the client ID obtained in the previous steps, agree to the terms, and click on *Register*.
+  13. Under *Add a featured channel*, click on *Microsoft Teams*, select your Microsoft Teams solution, click on *Save* and agree to the terms when prompted.
+  14. Navigate to Microsoft Teams and then to the *Apps* tab.
+  15. Search for the *Developer Portal* app and add it to your workspace.
+  16. Open the *Developer Portal*, click on *Apps* and then on *New app*.
+  17. Fill out the *Name* for the app and click on *Add*.
+  18. Navigate to *Basic information* and fill out the required fields: *Short description*, *Long description*, *Developer or company name*, *Website*, *Privacy policy*, *Terms of use*, and *Application (client) ID*. You may also provide any additional information you wish.
+  <Tip>
+  Please note that the above fields are required to be filled out for the bot to be usable in Microsoft Teams. The URLs provided in the *Website*, *Privacy policy*, and *Terms of use* fields must be valid URLs.
+  </Tip>
+
+  19. Navigate to *App features* and select *Bot*.
+  20. Choose the *Select an existing bot* option and select the bot you created earlier in the Microsoft Bot Framework portal.
+  21. Select all of the scopes where bot is required to be used and click on *Save*.
+  22. Finally, on the navigation pane, click on *Publish* and then on *Publish to your org*.
+
+For the bot to be made available to the users in your organization, you will need to ask your IT administrator to approve the submission at this [link](https://admin.teams.microsoft.com/policies/manage-apps). 
+
+Once it is approved, users can find it under the *Built for your org* section in the *Apps* tab. They can either chat with the bot directly or add it to a channel. To chat with the bot in a channel, type `@<bot_name>` followed by your message.
+
+While waiting for approval, you can test the chatbot by clicking on *Preview in Teams* in the Developer Portal.
+
```

Comment:
Let's also add the `Usage` section to show how to query tables.

Reply:
@martyna-mindsdb With the new implementation of this integration, there aren't really any tables to use as such. This is meant to be used solely as a chat handler (for chat-bots).
It should be possible to add tables, if required, but it would involve the same cumbersome authentication process our customer wanted to avoid.